nengo_bones/templates/static.sh.template

```diff
@@ -8,11 +8,11 @@ shopt -s globstar
 
 {% block install %}
 {{ super() }}
-    exe pip install "jupyter>=1.0.0" "pylint>=1.9.2,<2.5" "codespell>=1.12.0" "gitlint>=0.1.2" "collective.checkdocs>=0.2" "flake8>=3.7.7"
+    exe pip install "jupyter>=1.0.0" "pylint>=2.5.1" "codespell>=1.12.0" "gitlint>=0.1.2" "collective.checkdocs>=0.2" "flake8>=3.7.7"
 {% endblock %}
 
 {% block script %}
-    exe pylint {{ pkg_name }} --rcfile=setup.cfg --jobs=0
```

Comment:
I've got `pylint==2.5.3` on my machine, and I still need to use the `--rcfile` when I run things locally (I tried upgrading to 2.6.0, and same thing). If I run with `--verbose`, it says "No config file found", so somehow it's not detecting the pylint section in `setup.cfg`. (I'm testing with nengo, running from the project root folder.)

Reply:
Ahh, if I run `bones-generate` and get the `[pylint.*]` sections added (as added below), it uses it properly without `--rcfile`. I guess we don't have any projects that use the CI scripts without using nengo-bones to generate `setup.cfg`, so this should be fine. At the same time, I don't think there's any harm to being explicit and keeping the `--rcfile` in.

Reply:
Removing the explicit `--rcfile` allows the command to be run more flexibly (allows pylint to search for the `setup.cfg` file, rather than requiring it to be in the same directory). It was also kind of a hack the way we were doing it before (when `setup.cfg` wasn't supported by `pylint`, you could trick it into using one anyway through the `rcfile` option). Now that they officially support `setup.cfg`, we should always be using the correct configuration style (which has the `[pylint.*]` sections).